src/block/timeline/edit.js

```diff
@@ -298,11 +299,18 @@ const Edit = props => {
 		if ( nextBlock && nextBlock.name === 'stackable/timeline' && props.attributes.timelineIsLast ) {
 			dispatch( 'core/block-editor' ).__unstableMarkNextChangeAsNotPersistent()
 			setAttributes( { timelineIsLast: false } )
-		} else if ( ! nextBlock || nextBlock.name !== 'stackable/timeline' ) {
+			isUpdating.current = true
+		} else if ( ( ! nextBlock || nextBlock.name !== 'stackable/timeline' ) && props.attributes.timelineIsLast ) {
 			dispatch( 'core/block-editor' ).__unstableMarkNextChangeAsNotPersistent()
 			setAttributes( { timelineIsLast: true } )
+			isUpdating.current = true
 		}
-		updateMaxHeight()
+		if ( isUpdating.current ) {
+			updateMaxHeight()
+			setTimeout( () => {
+			  isUpdating.current = false
+			}, 300 )
+		  }
```

Comment:
When `nextBlock` changes quickly, the code will only run `updateMaxHeight()` for the first change. You may need to place it inside the `setTimeout` so it will be called at the very last change:
```suggestion
		if ( isUpdating.current ) {
			setTimeout( () => {
			  updateMaxHeight()
			  isUpdating.current = false
			}, 300 )
		  }
```